tests/new_functional/helpers.js

```diff
@@ -0,0 +1,99 @@
+const _ = require('lodash');
+const landingPage = require('./pages/landing.page');
+const generateToken = require('./jwt_helper');
+
+const getUri = uri => browser.options.baseUrl + uri;
+
+const getRandomString = length => _.sampleSize('ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789', length).join('');
+
+const startCensusQuestionnaire = (schema, sexualIdentity = false, region = 'GB-ENG', language = 'en') => {
+  return generateToken(schema, getRandomString(10), getRandomString(10), null, null, region, language, sexualIdentity)
+    .then(function(token) {
+      return browser.url('http://localhost:5000/session?token=' + token);
```

Comment:
Can you change these to `/session?token=' + token` the localhost bit must have sneaked in. But it means that it can't be run against remote environments